### PR TITLE
feat: capture arccontainerstorage in support bundle

### PR DESCRIPTION
**Pull request by Elsie4ever**
1. added ESA resource in support bundle, added `--svc acs` option
2. temperary skip test for orc, will address in the orc removal pr
image

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

**Comment by Elsie4ever** [comment]
IT runs
image

**Comment by Elsie4ever** [comment]
latest IT runs
image

File: azext_edge/edge/_help.py [hunk 1]
```diff
@@ -79,6 +81,10 @@ def load_iotops_help():
         - name: Include mqtt broker traces in the support bundle. This is an alias for stats trace fetch capability.
           text: >
             az iot ops support create-bundle --ops-service broker --broker-traces
+
+        - name: Include arccontainerstorage resources in the support bundle.
```
Comment (vilit1) [inline azext_edge/edge/_help.py:85]: no spaces in arccontainerstorage?

Comment (Elsie4ever) [reply azext_edge/edge/_help.py:85]: there will be, thanks!

File: azext_edge/edge/_help.py [hunk 2]
```diff
@@ -79,6 +81,10 @@ def load_iotops_help():
         - name: Include mqtt broker traces in the support bundle. This is an alias for stats trace fetch capability.
           text: >
             az iot ops support create-bundle --ops-service broker --broker-traces
+
+        - name: Include ARC container storage resources in the support bundle.
```
Comment (digimaun) [inline azext_edge/edge/_help.py:85]: Is Arc normally all caps ?

File: azext_edge/edge/providers/edge_api/arccontainerstorage.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from .base import EdgeResourceApi
+from ...common import ListableEnum
+
+
+class ArcContainerStorageResourceKinds(ListableEnum):
```
Comment (digimaun) [inline azext_edge/edge/providers/edge_api/arccontainerstorage.py:11]: This enum is only necessary for checks correct ?

File: azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        prefixes=[
+            "esa-otel-collector",
+            "csi-wyvern-controller",
+            "csi-wyvern-node",
+            "config-operator",
+            "edgevolume-mounthelper",
+            "w-adr-schema-registry",
+            "wyvern-operator",
+            "adr-schema-registry-cache-claimsrv",
+            "adr-schema-registry-cache-claim-user-pvc",
```
Comment (digimaun) [inline azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py:45]: Assuming these ADR elements are applicable only when schema registry is deployed?

**Review by digimaun — COMMENTED** [review]
We should think about refactoring / simplifying tests for SB.

**Review by digimaun — APPROVED** [review]

**Comment by Elsie4ever** [comment]
> We should think about refactoring / simplifying tests for SB.

got it, will do once we finish all the higher priority work item

File: azext_edge/edge/_help.py (hunk 2 above)
Comment (Elsie4ever) [reply azext_edge/edge/_help.py:85]: no, will update!

Comment (Elsie4ever) [reply azext_edge/edge/_help.py:85]: yeah will remove

File: azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py (hunk 4 above)
Comment (Elsie4ever) [reply azext_edge/tests/edge/support/create_bundle_int/test_arccontainerstorage_int.py]: update to only check those prefixes when the schema registry pods under AIO are detected

**Pull request merged**